src/category_theory/groupoid.lean

```diff
@@ -54,10 +55,13 @@ section
 variables {C : Type u} [category.{v} C]
 
 /-- A category where every morphism `is_iso` is a groupoid. -/
+noncomputable
 def groupoid.of_is_iso (all_is_iso : ∀ {X Y : C} (f : X ⟶ Y), is_iso f) : groupoid.{v} C :=
-{ inv := λ X Y f, (all_is_iso f).inv }
+{ inv := λ X Y f, inv f }
 
 /-- A category where every morphism has a `trunc` retraction is computably a groupoid. -/
+-- FIXME this has unnecessarily become noncomputable!
+noncomputable
 def groupoid.of_trunc_split_mono
   (all_split_mono : ∀ {X Y : C} (f : X ⟶ Y), trunc (split_mono f)) :
   groupoid.{v} C :=
```

Comment:
Ah, we probably should still fix this.

Reply:
I don't think this is needed for anything (I was going to use it to define free groupoids, but abandoned this idea).

More generally, if `f` is a morphism in some groupoid, then `inv f` comes from 
`is_iso` rather than `groupoid.iso`, so will always be given by
`classical.some` after this change, right? So why not remove the data from
`groupoid` and just let it say 'every morphism `is_iso`'?